### DISCUSSION TITLE
How to start Adonis server on subdomain in local development?

### DISCUSSION THREAD

**Discussion by feniljariwala82** [adonisjs/core#3150] [Help]
I am building an **SSO application** using adonis. And I need two servers running on example.com("Identity provider') and subdomain.example.com("Service provider") for testing purposes. Can anyone guide me on how to archive this? Or any suggestions are welcomed!

**Comment by mdjamilaj** [comment]
Use Laragon .
![image](https://user-images.githubusercontent.com/57256323/132947278-3d4b8a03-7b7c-4df1-ac56-16fc09a8fe38.png)
